website/src/docs/hotchocolate/fetching-data/index.md

```diff
@@ -1,289 +1,49 @@
+# Pagination
 
-```graphql
-query {
-  me {
-    name
-  }
-}
-```
+Hot Chocolate provides pagination capabilities out of the box. They allow us to expose pagination in a standardized way and can even take care of crafting the necessary pagination queries to our databases.
 
-In Hot Chocolate, this query results in the following resolver tree.
+[Learn more about pagination](/docs/hotchocolate/fetching-data/pagination)
 
-```mermaid
-graph TD
-  A(query: QueryType) --> B("me() => [UserType]")
-  B --> C("name() => StringType")
-```
+# Filtering
 
-A resolver tree is, in the end, nothing else than a resolver chain where each branch can be executed in parallel.
+When returning a list of entites, we often need to filter them using operations like `equals`, `contains`, `startsWith`, etc. Hot Chocolate takes away a lot of the boilerplate, by handling the generation of necessary input types and even translating the applied filters into native database queries.
 
-```mermaid
-graph LR
-  A("me()") --> B("name()")
-```
+[Learn more about filtering](/docs/hotchocolate/fetching-data/filtering)
 
-Okay, let's dissect a little further here. A resolver chain always starts with one or many root resolver, which is in our case `me()` and then follows the path along. In this scenario, the next resolver would be `name()`, which is also the last resolver in our chain. As soon as `me` has fetched the user profile of the currently logged-in user, Hot Chocolate will immediately start executing the next resolver and feeding in the previous object value, also called a parent or parent value in spec language. Let's say the parent value looks like this.
+# Sorting
 
-```csharp
-var parent = new User
-{
-  Id = "user-1",
-  Name = "ChilliCream",
-  ...
-}
-```
+Similar to filtering, Hot Chocolate can also autogenerate input types related to sorting. They allow us to specify by which fields and in which direction our entities should be sorted. These can also be translated into native database queries automatically.
 
-Then the `name()` resolver can just access the `Name` property of the parent value and simply return it. As soon as all resolvers have been completed, the execution engine would return the following GraphQL result, provided that everything went successful.
+[Learn more about sorting](/docs/hotchocolate/fetching-data/sorting)
 
-```json
-{
-  "data": {
-    "me": {
-      "name": "ChilliCream"
-    }
-  }
-}
-```
+# Projections
 
-Excellent, now that we know what resolvers are and how they work in a bigger picture, how can we start writing one. Let's jump to the next section and find out.
+Projections allow Hot Chocolate to project incoming GraphQL queries to the database.
 
-# Defining a resolver
+If we for example only select the `Name` and `Id` of a user in our GraphQL query, Hot Chocolate will only select those two fields when querying the user in the database.
```

Comment:
```suggestion
For example, if the client only requests the `name` and `id` of a user in their GraphQL query, Hot Chocolate will only query the database for those two columns.
```

Do you also want to mention it can handle relational properties?